vumi/tests/fake_connection.py

```diff
@@ -220,10 +220,14 @@ def patch_transport_abortConnection(transport, protocol):
     Patch abortConnection() onto the transport if it doesn't already have it.
     (`Agent` assumes its transport has this.)
     """
```

Comment:
The docstring should probably explain that even if the method is there, we may need to track that it's an abort so we can swap out the connectionLost reason.